src/proto/grpc/testing/stats.proto

```diff
@@ -41,6 +41,12 @@ message ServerStats {
   // change in server time (in seconds) used by the server process and all
```

Comment:
How is sum-of-squares useful? If we want to calc std deviation we could do it from the histogram using a median value for each bucket and the result would be pretty accurate.